engine/config.go

```diff
@@ -2,10 +2,15 @@ package engine
 
 import (
 	"encoding/json"
+	"bytes"	
 	"io"
 	"os"
+	
+	"github.com/ghodss/yaml"
```

Comment:
The package `github.com/ghodss/yaml` should be added to the `Gopkg.toml` file

Reply:
Added